src/components/Loader.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Loader = () => (
-  <div className="loader" />
+const Loader = ({ smaller }) => (
```

Comment:
Are all loaders in the system made smaller?

Reply:
No, so by default you will have the loader with classname loader. but in case you want a smaller loader like I did. You will need to pass a prop of smaller to the loader.